src/piping.ts

```diff
@@ -78,7 +78,7 @@ const NAME_TO_RESERVED_PATH = {
 };
 
 const indexPage: string =
-`<!DOCTYPE html>
+  `<!DOCTYPE html>
```

Comment:
Can you remove the white space? If you have some reasons to add the space, let me know.